api/app/outcome/api.py

```diff
@@ -51,9 +51,10 @@ class OutcomeAPI(restful.Resource):
     def get(self):
         req_parser = reqparse.RequestParser()
         req_parser.add_argument('event_id', type=int, required=True)
+        req_parser.add_argument('user_id', type=int, required=True)
         args = req_parser.parse_args()
 
-        user_id = g.current_user['id']
+        user_id = args['user_id']
```

Comment:
Is it an event admin that you're intending on using this API endpoint?

The reason we would have auto-set the `user_id` is likely because this is the endpoint that it's intended for the logged in user (in this context an applicant) to find out their outcome (and _only_ their outcome i.e. it's insecure to allow any user to be able query any other users outcome).

If indeed you were intending for the event admin to be able to query any outcome (from a user that's part of their event) then we would need to create another API endpoint (`OutcomeAdminAPI` maybe?) with a `get` method and the `@event_admin_required` decorator like what's on the `post` method. Let me know if you need help with this

Reply:
Yes, that makes sense! I was thinking this would be for the the event admin specifically and can make the change. I didn't see this endpoint being used elsewhere and was thinking of just updating it so that it's only for the admin. Would this make sense or should I still keep the regular endpoint and add the admin specific api.